lib/main.js

```diff
@@ -1,37 +1,38 @@
-"use strict";
+'use strict'
 
-var fs = require("fs");
+var fs = require('fs')
 
 module.exports = {
   /*
    * Main entry point into dotenv. Allows configuration before loading .env and .env.$NODE_ENV
-   * @param {Object} options - valid options: path (".env"), encoding ("utf8")
+   * @param {Object} options - valid options: path ('.env'), encoding ('utf8')
    * @returns {Boolean}
   */
-  config: function(options) {
-    var path = ".env",
-    encoding = "utf8";
+  config: function (options) {
```

Comment:
[`space-before-function-parentheses`](http://eslint.org/docs/rules/space-before-function-parentheses.html) would be a good rule to add to `.eslintrc`